### DISCUSSION TITLE
Android sms gateway using bluetooth

### DISCUSSION THREAD

**Discussion by cilinjohnrey** [ibnux/Android-SMS-Gateway#18] [Ideas]
Is it possible to use bluetooth to send sms instead of fcm?

**Comment by ibnux** [comment]
Yes its possible

But i don't plan to use Bluetooth

I never create Bluetooth project,it will take time for me to implement Bluetooth

**Reply by cilinjohnrey** [reply]
do you have any sources or references on how to implement? i will try to develop it. I have a goal to make android sms gateway offline by bluetooth and it can make a big impact to the community.

**Reply by ibnux** [reply]
You can use this library

https://github.com/OmarAflak/Bluetooth-Library

You need to create another apps to send Bluetooth sms

Or you can make app to listen on port 
So you can connect with wifi

https://github.com/piotrpolak/android-http-server

**Reply by cilinjohnrey** [reply]
https://github.com/OmarAflak/Bluetooth-Library about this library in sending a message from pc i will route to sms app in order to send sms?

**Reply by ibnux** [reply]
yes you need to create bluetooth listener and check PushService.java to learn how to send SMS

you can send bluetooth from browser

https://web.dev/bluetooth/
